app/models/subject.rb

```diff
@@ -139,9 +139,9 @@ def download_comments
 
   def github_client
     if app_installation.present?
-      Octobox.installation_client(app_installation.github_id)
+      app_installation.github_client
```

Comment:
This is just using a method that already exists!

Reply:
While this is existing I am not sure if I believe this is using the correct client! See my comment up in `Notification#github_client` for more context

Reply:
👍 for the deduplication